### PR TITLE
Add script file

### PR DESCRIPTION
**Pull request by Arafat-Anik**
Sample Descriptions

### CODE REVIEW

**Comment by hridoydas** [comment]
Do not practice with directly uploading a file

File: Script.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const chrome = require("selenium-webdriver/chrome");
+async function testRun() {
+    const productName = "Nike react phantom run flyknit 2";
+    let driver = await new Builder().forBrowser("chrome").build();
+    await driver.manage().window().maximize();
+    await driver.get("https://demo.evershop.io/");
+    // await driver.findElement(By.className("search-icon")).click();
+    await driver.findElement(By.xpath("//a[@class='search-icon']")).click();
+    await driver.findElement(By.xpath("//input[@placeholder='Search']")).sendKeys("nike");
+    // await driver.findElement(By.xpath("//input[@placeholder='Search']")).sendKeys("nike",Key.ENTER);
```
Comment (hridoydas) [inline Script.js:11]: remove comment code

File: Script.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const { Builder, Browser, By, Key, until } = require('selenium-webdriver')
+const chrome = require("selenium-webdriver/chrome");
+async function testRun() {
+    const productName = "Nike react phantom run flyknit 2";
+    let driver = await new Builder().forBrowser("chrome").build();
+    await driver.manage().window().maximize();
+    await driver.get("https://demo.evershop.io/");
+    // await driver.findElement(By.className("search-icon")).click();
```
Comment (hridoydas) [inline Script.js:8]: remove commented code

**Review by hridoydas — CHANGES_REQUESTED** [review]

File: Script.js (hunk 1 above)
Comment (Arafat-Anik) [reply Script.js:11]: Done

**Review by hridoydas — APPROVED** [review]

**Pull request merged**